### PR TITLE
Simnet v8 move tests

### PR DESCRIPTION
**Pull request by radupopa2010**
Here is link to green pipeline: https://gitlab.parity.io/parity/polkadot/-/jobs/1068372
I hard-coded version name to v8, just trigger a real test.

Edited a bit the publishing jobs of needed container images for simnet tests; I think they are a little bit easier to understand and they work as expected: 

```
$ echo "Simnet Tests Config ${SIMNET_IMAGE_NAME} ${SIMNET_REF} ${PARACHAINS_IMAGE_NAME} ${PARACHAINS_IMAGE_TAG} ${COLLATOR_IMAGE_NAME} ${COLLATOR_IMAGE_TAG}"
Simnet Tests Config docker.io/paritytech/simnet v7 docker.io/paritypr/synth-wave 3647-0.9.9-16dd4740-5d496a12 docker.io/paritypr/colander 3647-16dd4740
```

check out: https://gitlab.parity.io/parity/polkadot/-/jobs/1068372

On this line:  https://github.com/paritytech/polkadot/blob/16dd4740596d45c15363c74564b392d7a51007fa/.gitlab-ci.yml#L487
and this line https://github.com/paritytech/polkadot/blob/16dd4740596d45c15363c74564b392d7a51007fa/.gitlab-ci.yml#L518
I could not use the variable `${SIMNET_IMAGE_NAME}` defined in the variables section of the `simnet-tests` jobs, because there was not value populated. We might define this variable in another place before the job definition or we use the hard-coded  value 
`docker.io/paritytech/simnet`

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -491,18 +494,29 @@ trigger-simnet:
...
-  # `build.env` brings here `$IMAGE_NAME` and `$IMAGE_TAG` (`$EXTRATAG` here,
-  # i.e. `2643-0.8.29-5f689e0a-6b24dc54`).
-  # `collator.env` bears adder-collator unique build tag. In non-triggered builds it
-  # can be called by `master` tag.
-  # Simnet uses an image published on PRs with this exact version for triggered runs
-  # on commits. And parity/rococo:rococo-v1 for the runs not launched by this job.
+  # `parachains.env` brings here `$PARACHAINS_IMAGE_NAME` and `$PARACHAINS_IMAGE_TAG` 
+  # (`$EXTRATAG` here, # i.e. `2643-0.8.29-5f689e0a-6b24dc54`).
+  # `collator.env` brings here `$COLLATOR_IMAGE_NAME` and  `$COLLATOR_IMAGE_TAG` 
+  # In non-ci jobs `$PARACHAINS_IMAGE_TAG` and `$COLLATOR_IMAGE_TAG`
```
Comment (TriplEight) [inline .gitlab-ci.yml:500]: what's a "non-ci job"?

File: .gitlab-ci.yml [hunk 2]
```diff
@@ -491,18 +494,29 @@ trigger-simnet:
...
+        ${COLLATOR_IMAGE_NAME} ${COLLATOR_IMAGE_TAG}"
   script:
-    # API trigger for a Simnet job, argument value is set in the project variables
-    - ./scripts/gitlab/trigger_pipeline.sh --simnet-version=${SIMNET_REF}
+    - /home/nonroot/simnet/gurke/scripts/run-test-environment-manager.sh
+        --github-remote-dir=https://github.com/paritytech/polkadot/tree/master/simnet_tests
+        --config=simnet_tests/configs/simple_rococo_testnet.toml
+        --image="${PARACHAINS_IMAGE_NAME}:${PARACHAINS_IMAGE_TAG}"
+        --image-2="${COLLATOR_IMAGE_NAME}:${COLLATOR_IMAGE_TAG}"
+        --image-simnet="docker.io/paritytech/simnet:v8"
```
Comment (TriplEight) [inline .gitlab-ci.yml:518]: FIXME: v8

File: .gitlab-ci.yml [hunk 3]
```diff
@@ -479,9 +481,10 @@ deploy-polkasync-kusama:
   allow_failure:                   true
   trigger:                         "parity/infrastructure/parity-testnet"
 
-trigger-simnet:
+simnet-tests:
   stage:                           deploy
-  image:                           paritytech/tools:latest
+  # fixme: revert to variable
+  image:                           "docker.io/paritytech/simnet:v8"
```
Comment (TriplEight) [inline .gitlab-ci.yml:487]: FIXME: v8

File: .gitlab-ci.yml [hunk 4]
```diff
@@ -491,18 +494,29 @@ trigger-simnet:
...
+  before_script:
+    - echo "Simnet Tests Config
+        ${SIMNET_IMAGE_NAME} ${SIMNET_REF}
+        ${PARACHAINS_IMAGE_NAME} ${PARACHAINS_IMAGE_TAG}
+        ${COLLATOR_IMAGE_NAME} ${COLLATOR_IMAGE_TAG}"
   script:
-    # API trigger for a Simnet job, argument value is set in the project variables
-    - ./scripts/gitlab/trigger_pipeline.sh --simnet-version=${SIMNET_REF}
+    - /home/nonroot/simnet/gurke/scripts/run-test-environment-manager.sh
+        --github-remote-dir=https://github.com/paritytech/polkadot/tree/master/simnet_tests
```
Comment (TriplEight) [inline .gitlab-ci.yml:514]:
```suggestion
        --github-remote-dir="https://github.com/paritytech/polkadot/tree/master/simnet_tests"
```

File: .gitlab-ci.yml [hunk 5]
```diff
@@ -491,18 +494,29 @@ trigger-simnet:
...
+        --config=simnet_tests/configs/simple_rococo_testnet.toml
```
Comment (TriplEight) [inline .gitlab-ci.yml:515]:
```suggestion
        --config="simnet_tests/configs/simple_rococo_testnet.toml"
```

**Review by TriplEight — APPROVED** [review]
Remove some nits and FIXMEs and good to go.

File: .gitlab-ci.yml (hunk 1 above)
Comment (radupopa2010) [reply .gitlab-ci.yml:500]: like local test with Docker

File: .gitlab-ci.yml [hunk 6]
```diff
@@ -491,18 +493,26 @@ trigger-simnet:
...
-  # i.e. `2643-0.8.29-5f689e0a-6b24dc54`).
-  # `collator.env` bears adder-collator unique build tag. In non-triggered builds it
-  # can be called by `master` tag.
-  # Simnet uses an image published on PRs with this exact version for triggered runs
-  # on commits. And parity/rococo:rococo-v1 for the runs not launched by this job.
-  variables:
-    TRGR_PROJECT:                  ${CI_PROJECT_NAME}
-    TRGR_REF:                      ${CI_COMMIT_REF_NAME}
-    # Simnet project ID
-    DWNSTRM_ID:                    332
```
Comment (TriplEight) [inline .gitlab-ci.yml:504]: maybe it's also appropriate to remove the corresponding parts in the triggering script? Or the entire triggering script.

**Review by TriplEight — CHANGES_REQUESTED** [review]

File: .gitlab-ci.yml (hunk 6 above)
Comment (radupopa2010) [reply .gitlab-ci.yml:504]: I removed it

**Review by TriplEight — APPROVED** [review]
LGTM, just updated a variable to `v8`.
Please merge after simnet test gets green.

**Pull request merged**